src/sentry/tagstore/snuba/backend.py

```diff
@@ -764,7 +769,7 @@ def get_tag_value_paginator_for_projects(
                 # user.alias is a pseudo column in discover. It is computed by coalescing
                 # together multiple user attributes. Here we get the coalese function used,
                 # and resolve it to the corresponding snuba query
-                dataset = Dataset.Discover
+                # dataset = Dataset.Discover
```

Comment:
Nit: Remove?

Reply:
Thanks. I noticed we might have a similar problem with TSDB too - will follow up on that one.